config/routes.rb

```diff
@@ -764,7 +764,7 @@ def call(env)
           resources :pricing_rules, :only => [:edit, :update, :destroy]
         end
 
-        resources :services, except: :index do
+        resources :services do
```

Comment:
This `services#index` also needs a test 😄

Reply:
Yes I'm not sure where to add it 🤷‍♂️